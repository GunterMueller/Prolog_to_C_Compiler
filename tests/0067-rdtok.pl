main :-
	see('tests/0067-rdtok.pl'),
	read_tokens(TOKS, VARS),
	write(TOKS), nl,
	write(VARS), nl,
	seen.
