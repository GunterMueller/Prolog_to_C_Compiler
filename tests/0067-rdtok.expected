[atom(main),atom((:-)),atom(see),(,atom(tests/0067-rdtok.pl),),(,),atom(read_tokens),(,var(_274,TOKS),(,),var(_331,VARS),),(,),atom(write),(,var(_274,TOKS),),(,),atom(nl),(,),atom(write),(,var(_331,VARS),),(,),atom(nl),(,),atom(seen)]
[TOKS = _274,VARS = _331]
